### ISSUE TITLE
Mathjax right-click menu is obstructed by cells

### ISSUE THREAD

**Issue by david-cortes** [microsoft/vscode#142989]
## Environment data

-   VS Code version: 1.63.2
-   Jupyter Extension version (available under the Extensions sidebar): v2021.11.1001550889
-   Python Extension version (available under the Extensions sidebar): v2022.0.1814523869
-   OS (Windows | Mac | Linux distro) and version: linux (debian sid)
-   Python and/or Anaconda version: conda 4.11.0
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local

## Steps to reproduce

* Lauch an interactive jupyter interpreter.
* Print something that would render as mathjax.
* Run some command so that it would show in the panel right below the cell with mathjax.
* Right-click the mathjax.

## Expected behaviour

Should show its right-click menu, superimposing it over the jupyter cell below so that the full right-click menu is visible and clickable.

## Actual behaviour

The right-click menu is rendered **below** the jupyter cell, thus making a big chunk of it invisible and unclickable.

![vscode_hidden_mathjax](https://user-images.githubusercontent.com/11860098/153800112-3ed68781-0bf2-4e44-85a9-a3f6090d03cd.png)

**Comment by ickc** [comment]
Copyable code:

```py
from sympy import symbols, exp
a = symbols("a")
x = symbols("x")
b = 1/ (1 + exp(-1 / exp(x)))
b
```

On my platform, it can't even be right clicked and show the MathJax menu:

Version: 1.64.2
Commit: f80445acd5a3dadef24aa209168452a3d97cc326
Date: 2022-02-09T22:00:58.347Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.3.0

**Comment by DonJayamanne** [comment]
@rebornix assigning to you as this was assigned to me when this was transferred.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22bug%22). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
